src/pages/certificates/certificate/certificate.styles.js

```diff
@@ -1,15 +1,42 @@
 import { makeStyles } from '@material-ui/core/styles';
+import certificate from '../../../assets/images/certificate.png';
 
 export const useStyles = makeStyles(() => ({
   wrapper: {
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center'
   },
+  certificate: {
+    width: '136px',
+    height: '63px',
+    backgroundImage: `url(${certificate})`,
```

Comment:
why didn't you use the single background rule?

Reply:
I used empty certificate without text. And for each certificate I added text with value. In this case we download only one image instead of three certificate with different value.

Reply:
I'm asking about combining background-image, background-cover etc to single background rule

Reply:
Fixed

Reply:
Isn't it better? )